apis/fluentbit/v1alpha2/clusterfluentbitconfig_types_test.go

```diff
@@ -246,8 +254,30 @@ func Test_FluentBitConfig_RenderMainConfig(t *testing.T) {
 		},
 	}
 
+	openSearchOutput := ClusterOutput{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "fluentbit.fluent.io/v1alpha2",
+			Kind:       "Output",
```

Comment:
Kind shoud be ClusterOutput?

Reply:
Yes, there should be `ClusterOutput`. I noticed that there are many place used `Output`, `Input` or `Filter` in `fluent-bit config`, seems they should be also need add a `Cluster` prefix, and do we need solve also in this PR?

Reply:
Yes. Pls help to correct it.

Reply:
The kind should always be clusterxxx for fluentbit crds like this https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/clusterinput_types.go#L52

But we needn't change the internal structure to clusterxxx like this: https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/clusterinput_types.go#L34

Reply:
input, output, filter parser CRDs are all changed to clusterxxx in v1.0.0

Reply:
Done.